15_Ansible_test/ansible_roles_test/db_tasks_run_role.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Database tasks
+  hosts: remote_admin
+  become: yes
+  become_user: admin
+  remote_user: admin
```

Comment:
it's not required to define here a remote user, because if have done it in the inventory file.